custom_components/tuya_local/devices/klarstein_wonderwall_smart.yaml

```diff
@@ -0,0 +1,54 @@
+name: Klarstein Wonderwall Smart
+products:
+  - id: iuahvrtxktwuuyaf
+    name: Wonderwall Smart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
```

Comment:
Mapping can be removed, scale and step of 1 are the default and no not need to be listed.

Reply:
Suggest specifying `unit: C` under temperature otherwise it will follow the HA user settings, which might not match the device.

Reply:
File removed in deference to klarstein_wonderwall_irheater.yaml added under issue #1272